esctest/tests/ris.py

```diff
@@ -49,6 +49,8 @@ def test_RIS_ResetTitleMode(self):
     esccmd.ChangeIconTitle("a")
     AssertEQ(GetIconTitle(), "a")
 
+  @vtLevel(4)
+  @knownBug(terminal="iTerm2", reason="iTerm2 doesn't support ALTBUF.")
```

Comment:
iTerm2 does support ALTBUF. Did you find a bug in it?

Reply:
I overlooked the change when I prepared the branch (recall that I started from the iTerm2 repo).